src/mbgl/map/map.cpp

```diff
@@ -236,6 +236,10 @@ void Map::cancelTransitions() {
     impl->onUpdate(Update::Repaint);
 }
 
+void Map::setCameraMutated(bool cameraMutated) {
```

Comment:
I think this is a bit obscure for a public API.

I would rather have an API for resetting the camera to the style defaults, something like `Map::resetCameraToStyleDefaults`